### ISSUE TITLE
Error using download_analysis_data.py

### ISSUE THREAD

**Issue by xylar** [MPAS-Dev/MPAS-Analysis#386]
I saw the following error when I was downloading the obs. data:
```
Exception in thread Thread-4:
Traceback (most recent call last):
  File "/home/xylar/miniconda3/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/home/xylar/miniconda3/lib/python3.6/threading.py", line 864, in run
    self._target(*self._args, **self._kwargs)
  File "./download_analysis_data.py", line 84, in download_worker
    urlretrieve('{}/{}'.format(urlBase, fileName), outFileName)
  File "/home/xylar/miniconda3/lib/python3.6/urllib/request.py", line 289, in urlretrieve
    % (read, size), result)
urllib.error.ContentTooShortError: <urlopen error retrieval incomplete: got only 3997696 out of 94362664 bytes>
```
The script never finished as a result of the error.  Seems like it got stuck somehow.

From searching on the web, it seems like `request` might be a better package than `urllib` for downloading files.  I will look into this.

**Comment by milenaveneziani** [comment]
hmm, OK, let me know what you find.
Do you think it's a random error?

**Comment by xylar** [comment]
Most likely a random error but it might still be best to move to something more robust if that's what `request` provides.

**Comment by xylar** [comment]
I believe this will be addressed by #392
